src/App.jsx

```diff
@@ -14,45 +16,95 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const sortByLength = 'byLength';
+const sortByAlphabet = 'ByAlphabet';
+
+function sortGoods(goods, { sortValue, reverse }) {
+  const sortedGoods = [...goods];
+
+  if (sortValue) {
+    sortedGoods.sort((good1, good2) => {
+      switch (sortValue) {
+        case sortByLength:
+          return good1.length - good2.length;
+        case sortByAlphabet:
+          return good1.localeCompare(good2);
+        default:
+          return 0;
+      }
+    });
+  }
+
+  return reverse ? sortedGoods.reverse() : sortedGoods;
+}
+
+export const App = () => {
+  const defaultSortSettings = { sortValue: '', reverse: false };
```

Comment:
1. move default state outside the component
2. change a key name a little
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

```suggestion
const defaultSortSettings = { sortValue: '', isReversed: false };

export const App = () => {
```